Formula/cddlib.rb

```diff
@@ -0,0 +1,32 @@
+class Cddlib < Formula
+  desc "Double description method for general polyhedral cones"
+  homepage "https://www.inf.ethz.ch/personal/fukudak/cdd_home/"
+  url "ftp://ftp.math.ethz.ch/users/fukudak/cdd/cddlib-094h.tar.gz"
+  sha256 "fe6d04d494683cd451be5f6fe785e147f24e8ce3ef7387f048e739ceb4565ab5"
+
+  depends_on "gmp"
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+    doc.install Dir["doc/*"]
+    pkgshare.install Dir["examples*"]
+  end
+
+  test do
+    (testpath/"test.cpp").write <<~EOS
+      #include "setoper.h"
+      #include "cdd.h"
+
+      int main(int argc, char *argv[])
+      {
+        dd_set_global_constants(); /* First, this must be called once to use cddlib. */
+        printf("Welcome to cddlib %s\n",dd_DDVERSION);
+        return -1
+      }
+    EOS
+    system ENV.cxx, "test.cpp", "-I#{include}/cddlib", "-o", "test"
+    system "./test"
```

Comment:
Can you use `shell_output` and `assert_match` here to check for some output?

Reply:
I will.

Reply:
OK. This is done!